src/internal/cmdutil/cobra.go

```diff
@@ -319,7 +319,17 @@ func (r *RepeatedStringArg) Type() string {
 // '{{alias}}' will be replaced with the full command path.  These commands can
 // later be merged into the final Command tree using 'MergeCommands' below.
 func CreateAlias(cmd *cobra.Command, invocation string) *cobra.Command {
+	return createAlias(cmd, invocation)
+}
+
+// CreateAliases is like CreateAlias, except it allows us to specify one or more plurals for the
+// last argument in the command with the assumption that the last argument in the command is a resource
+// such as 'repo' or 'pipeline'.
+func CreateAliases(cmd *cobra.Command, invocation string, plurals ...string) *cobra.Command {
```

Comment:
maybe synonyms > plurals is a more accurate parameter name? WDYT?

Reply:
Yeah I think that makes more sense. This way you could do shorthand versions to like `fi` for `files` or something